packages/button/src/button.tsx

```diff
@@ -44,15 +44,19 @@ export interface ButtonContentProps {
   leftIcon?: ButtonProps['leftIcon'];
   rightIcon?: ButtonProps['rightIcon'];
   children?: ButtonProps['children'];
+  size?: ButtonProps['size'];
 }
 
 export const ButtonContent: React.FC<ButtonContentProps> = (props: ButtonContentProps): JSX.Element => {
-  const { loading, loadingText, leftIcon, rightIcon, children } = props;
+  const { size = 'md', loading, loadingText, leftIcon, rightIcon, children } = props;
   return (
     <>
       {leftIcon && !loading ? <span className="btn-has-icon-left">{leftIcon}</span> : null}
       {loading && (
-        <Spinner className={cx(loadingText ? 'relative' : 'absolute', loadingText ? `mr-2` : 'mr-0')} size={16} />
+        <Spinner
+          size={size === 'sm' ? 1.6 : size === 'md' ? 1.8 : 2}
+          className={cx(loadingText ? 'relative' : 'absolute', loadingText ? `mr-2` : 'mr-0')}
```

Comment:
Would it not be better to add a "sk-button-spinner-hasLoadingText"-class or such instead of adding specific tailwind-classes, to separate the styling?

Reply:
Yes, that is not part of this fix however